clientgui/AdvancedFrame.cpp

```diff
@@ -2118,8 +1959,7 @@ void CAdvancedFrame::OnNotebookSelectionChanged(wxNotebookEvent& event) {
     wxLogTrace(wxT("Function Start/End"), wxT("CAdvancedFrame::OnNotebookSelectionChanged - Function End"));
 }
 
-
-void CAdvancedFrame::ResetReminderTimers() {
+void CBOINCBaseFrame::ResetReminderTimers() {
```

Comment:
I believe this should be moved to clientgui/BOINCBaseFrame.cpp

Reply:
Done.